config/checkstyle_resources_checks.xml

```diff
@@ -25,7 +25,7 @@
     <property name="folderPattern"
              value="[\\/]src[\\/](it|test)[\\/]resources(-noncompilable?)[\\/]"/>
     <property name="fileNamePattern"
-             value="^(package-info.java)|(Input\w+\.java)|(.*\.properties)$"/>
+             value="^(package-info.java)|(Input\w+\.java)|(Input\w+\.txt)|(.*\.properties)$"/>
```

Comment:
Existing pattern did not allow for expected AST output files in non-compilable. I think it is better to keep input files and expected output together.